index.html

```diff
@@ -34,7 +34,11 @@
       <nav>
         <div class="header-grid">
           <div class="header-title">
-            <h2>Tutor Me</h2>
+            <img
+              class="logo"
+              src="C:\Users\achri\OneDrive\Desktop\Coding Projects\Portfolio\TutorMe\v29-toucans-team-01\images\3.png"
```

Comment:
The logo doesn't seem to be showing in my view.  Not sure if everyone else is seeing it.  
![image](https://user-images.githubusercontent.com/42394859/118074077-3e269f80-b37b-11eb-8b8c-1d1ab30949a7.png)

Reply:
Yes I had the same issue when I did the initial pull but I had to go back and change the file name again, similar to the issue I had before with the css and js,